libraries/default.rb

```diff
@@ -0,0 +1,68 @@
+def collect_github_repositories(github_token, org)
+  if ::ObjectSpace.const_defined?('Chef')
+    # Chef exists! Lets make sure Octokit is installed.
+    chef_gem 'octokit'
+  end
```

Comment:
I see what you mean. In that case we don't want to install gems for nodes using the client recipe. We can keep the requires in the functions but it would still be good to move them the `lib/github.rb` and such. For chef_gem just make sure its getting called before those functions and everything should be fine.